app/helpers/sac_cas/people_helper.rb

```diff
@@ -21,12 +21,16 @@ def format_person_sac_family_main_person(person)
   end
 
   def qr_code_image_tag(person, html_options = {})
```

Comment:
Maybe the naming is now slightly confusing, since its a link tag containing an image tag?

Reply:
the thing I don't like about helpers in rails that they share the same namespace. so we always have to be careful how to name things. 
anyway ... how about renaming this to `people_sac_membership_qr_code` ...
this way it would be prefixed with people, belongs to membership feature and returns the qr code